driver-core/src/test/java/com/datastax/driver/core/ProtocolV1Test.java

```diff
@@ -24,7 +25,7 @@
 /**
  * Tests targeting protocol v1 specifically.
  */
-@CCMConfig(version = "1.2.19")
+@CassandraVersion(major = 1.2, exactRequired = true)
```

Comment:
Oh. So we need OS requirements too, I will look into that for the new test framework. 

Ideally, we would have to amend `@CCMConfig` to add a new attribute, e.g. `os = {NIX, MAC_OS}` and have the test skipped/ignored if the os doesn't match.

But that's complex, so in the meanwhile I think that what you currently have done so far is enough.